src/interceptors/ClientRequest/NodeClientRequest.ts

```diff
@@ -20,6 +20,10 @@ import { toInteractiveRequest } from '../../utils/toInteractiveRequest'
 import { uuidv4 } from '../../utils/uuid'
 import { emitAsync } from '../../utils/emitAsync'
 
+const kHeadersList = Object.getOwnPropertySymbols(new Headers()).find(
```

Comment:
I recommend highly against tapping into Node's internal symbols. They are tedious to work with intentionally—you shouldn't access them unless you own them. They can be swapped at any point of time in Node, and more so across different versions, making them a big maintenance issue.